submissions/EmmanuelIdiaghe/php-database/admin.php

```diff
@@ -0,0 +1,72 @@
+<?php
+#Variable Declaration
+$notSet = !(isset($_COOKIE["show"])); //cookie not set
+
+
+# Function that sets cookie
+function load() {
+    //SetUp Connection
+    global $pdo;
+    $usernameDB = "dufuna";//user
+    $passwordDB = "dufuna12345";//user password
+
+    try {
+        $pdo = new PDO("mysql:host=localhost", $usernameDB, $passwordDB);
+        $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+        echo "Connected Successfully. Host info: " . $pdo->getAttribute(constant("PDO::ATTR_CONNECTION_STATUS")) . "<br>";
+    
+    } catch (PDOException $e) {
+        die ("ERROR: Could not connect. " . $e->getMessage()) . "<br>";
+    }
+    
+    #Use Database
+    try {
+        $sql = "USE emmy_shop";
+        $pdo->exec($sql);
+        echo "Using Database: emmy_shop <br><hr><br>";
+    
+    } catch (PDOException $e) {
+        die ("ERROR: Could not use database $sql " . $e->getMessage()) . "<br>";
+    }
+    
+    $sql = "SELECT * FROM customers";
+    $result = $pdo->query($sql);
+    $details = $result->fetchAll();
+    setcookie("show", serialize($details), time() + (86400 * 30), "/");
+    $_COOKIE["show"] = serialize($details);
```

Comment:
I think line 35 and 36 almost do the same thing

Reply:
Yes, they actually do the same thing except that the second forces the cookie to set immediately.

Reply:
I think you need to use just the `setcookie` method

Reply:
Also, this function should return something